### PR TITLE
Update user facing messages to reference 'agent binaries' instead of 'tools'

### PR DESCRIPTION
**Pull request by hmlanigan**
## Please provide the following details to expedite Pull Request review:

----

## Description of change

This is a follow on to https://github.com/juju/juju/pull/7354.  It was mentioned in general
to use 'agent binaries' or the like instead of 'tools'.  Updating user facing output accordingly.

## QA steps

1. Verify visually during juju bootstrap --debug and in logs
2. Ran make check in juju/juju

## Documentation changes

Unknown, it may affect some documentation examples?

### CODE REVIEW

File: environs/sync/sync.go [hunk 1]
```diff
@@ -159,7 +159,7 @@ func selectSourceDatasource(syncContext *SyncContext) (simplestreams.DataSource,
 	if err != nil {
 		return nil, err
 	}
-	logger.Infof("using sync tools source: %v", sourceURL)
+	logger.Infof("using sync agent binary source: %v", sourceURL)
```
Comment (jameinel) [inline environs/sync/sync.go]: maybe: using agent binary sync source
Neither one reads particularly well.

Comment (hmlanigan) [inline environs/sync/sync.go]: i went with 'source for sync of agent binaries:'

**Comment by hmlanigan** [comment]
$$merge$$

**Comment by hmlanigan** [comment]
$$merge$$

**Comment by hmlanigan** [comment]
$$merge$$

**Pull request merged**